battles/views.py

```diff
@@ -0,0 +1,66 @@
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.messages.views import SuccessMessageMixin
+from django.views.generic.edit import CreateView
+from django.views.generic.list import ListView
+
+from battles.forms import CreateBattleForm, SelectTrainerTeamForm
+from battles.models import Battle, TrainerTeam
+
+
+class CreateBattleView(
+    LoginRequiredMixin,
+    SuccessMessageMixin,
+    CreateView
+):
```

Comment:
I don't bother with this, but just to let you know if you have a future reviewer that bothers... We usually  define class multiline signatures like this:
```python
class CreateBattleView(
        LoginRequiredMixin, SuccessMessageMixin, CreateView, LoginRequiredMixin, 
        SuccessMessageMixin, CreateView):  
    pass
```

Reply:
Done!